main.py

```diff
@@ -259,19 +270,25 @@ def single_argument_handler(arg):
         slideshow(db, 387, 494, rand=arg.startswith("rnd"))
     elif arg == "?":
         print("This function is deprecated.")
-    elif escape_code:
+    elif ESCAPE_CODE:
         change_wallpaper(db, arg)
     else:
         change_terminal_background(db, arg)
 
-
 if __name__ == "__main__":
     # Entrance to the program.
     if len(argv) == 1:
         print('No command line arguments specified.'
               '\nTry typing in a Pokemon name or number.'
               '\nOr type "help" to see all the commands.')
-    elif len(argv) == 2:
+        sys.exit(1)
+    # If there is an escape code, then change the wallpaper, not the terminal.
+    if str(argv[1]).startswith("_"):
+        ESCAPE_CODE = True
```

Comment:
Why move backwards to defining a global constant?  Main needs to be a separate function so we can do end-to-end tests and this breaks that.